lib/fog/compute/google/models/images.rb

```diff
@@ -68,6 +68,25 @@ def get(identity)
           new(data)
         end
 
+        def get_from_family(family)
+          # Search own project before global projects
+          all_projects = [service.project] + global_projects
```

Comment:
I thought the suses were missing but they seem to be there, so never mind :p 

I don't know if it needs to be private, but yes.